otwcode/otwarchive
# AO3-6006 Show "pseud (username)" in dashboard heading

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6006

## Purpose

Make it clear on user dashboard that a pseud is being displayed by showing the username as well if a non-default pseud is being displayed.  Note that I have removed the links from the heading here.  I felt they weren't useful since you were already viewing this particular user's page.


## Testing Instructions

1. Go to user page of user with extra pseuds
2. Verify that if default pseud is selected only username is displayed in the heading
3. Verify that if a different pseud is selected, the byline format "pseud (username)" is displayed in the heading

New test added in `user_dashboard.feature` to account for this, note that I added examples to also check works and bookmarks pages for the same thing (didn't see tests for this elsewhere).  Also, this can easily be expanded with one more example for the series page (AO3-6007).

## References

See also AO3-6007 and #3887.  Same scenario outline is used for testing, since the expected outcome is similar.  There may be conflicts with that one depending on the merge order.

## Credit

tlee911 (he/him)
